platform-hub-web/src/app/projects/projects-form.html

```diff
@@ -49,6 +51,14 @@
             </textarea>
           </md-input-container>
 
+          <md-input-container class="md-block">
+            <label for="cost_centre_code">Cost centre code:</label>
+            <input
```

Comment:
Sure. Thought we could lay the foundations for that now and have that ready but can do later as well.